blockchain/README.md

```diff
@@ -0,0 +1,27 @@
+# Blockchain Template
+
+This Microsoft Azure Resource Manager template deploys a single VM and offers a variety of blockchain software to run on that VM.
```

Comment:
reword?
This Microsoft Azure Resource Manager template deploys a single VM and allows you to select from a number of blockchain software options to run on the VM.